### PR TITLE
At/#1423 reload settings and poll for KYC status

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
Polling for KYC status change, for both individual and business.

I couldn't reproduce the situation that the settings don't reload sol I'll leave it like this for now

### CODE REVIEW

File: app/components/settings/kyc-states/KycStatusWidget.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ export type IKycStatusWidgetProps = IStateProps & IDispatchProps & IOwnProps;
 const statusTextMap: UnionDictionary<TRequestStatus, React.ReactNode> = {
   Accepted: <FormattedMessage id="settings.kyc-status-widget.status.accepted" />,
   Rejected: <FormattedMessage id="settings.kyc-status-widget.status.rejected" />,
+  Ignored: <FormattedMessage id="settings.kyc-status-widget.status.rejected" />,
```
Comment (trival) [inline app/components/settings/kyc-states/KycStatusWidget.tsx]: is the translation id also rejected for "Ignored" state?

File: app/components/settings/Settings.tsx [hunk 2]
```diff
@@ -111,6 +111,7 @@ export const Settings = compose<React.SFC>(
   }),
   onEnterAction({
     actionCreator: dispatch => {
+      // dispatch(actions.kyc.kycLoadBusinessData())
```
Comment (Moejoe90) [inline app/components/settings/Settings.tsx]: There is a comment here

File: app/lib/api/KycApi.interfaces.ts [hunk 3]
```diff
@@ -108,7 +108,13 @@ export const KycFileInfoShape = Yup.object().shape({
 });
 
 // request state
-export type TRequestStatus = "Draft" | "Pending" | "Outsourced" | "Rejected" | "Accepted";
```
Comment (Moejoe90) [inline app/lib/api/KycApi.interfaces.ts]: I would change this to an enum as per our guidelines but this is not blocking

File: app/modules/kyc/sagas.ts [hunk 4]
```diff
@@ -54,12 +57,18 @@ function* kycRefreshWidgetSaga(): any {
...
+        outsourcedStatus &&
+        (outsourcedStatus === "started" ||
+          outsourcedStatus === "canceled" ||
+          outsourcedStatus === "aborted" ||
+          outsourcedStatus === "review_pending"))
     ) {
-      yield put(actions.kyc.kycLoadIndividualRequest(true));
+      requestType === "individual"
+        ? yield put(actions.kyc.kycLoadIndividualRequest(true))
+        : yield put(actions.kyc.kycLoadBusinessRequest());
```
Comment (Moejoe90) [inline app/modules/kyc/sagas.ts]: I'm not sure why does load individual request have true as input i would investigate what this means and if load business request requires true or not
Maybe you did this already

**Pull request merged**